Bottom: 893a94890deade6921e1fb95ec493f471d546d3b
Top:    a9e98367bf16bb8320a5c7e8a3fe91b52907ebaf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-04 01:24:17 +0000

Refresh of add-routines-to-render-just-a-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 93b27c2..dafa059 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -583,6 +583,7 @@ do_hole (const BoxType *b, void *cl)
 {
   struct do_hole_info *info = cl;
   PLINE *curc = (PLINE *) b;
+  cairo_traps_t *traps;
 
   /* Ignore the outer contour - we draw it first explicitly*/
   if (curc->Flags.orient == PLF_DIR) {
